### PR TITLE
small refactoring, conventional POSIX-compatible cli lib, convinient …

### PR DESCRIPTION
**Pull request by kirillDanshin**
…log library, systemd and docker configs

Signed-off-by: Kirill Danshin <k@guava.by>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -82,24 +89,28 @@ func checkMessage(m *tb.Message) {
 
 func silenceCommand(m *tb.Message) {
 	if !isAdmin(m.Chat, m.Sender) {
+		log.Debug("unauthorized message, removing")
```
Comment (stek29) [inline main.go]: logging at least m.Chat.ID & m.ID might be a good idea

File: README.md [hunk 2]
```diff
@@ -27,10 +27,23 @@ CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo -ldflags '-w -extl
...
 ## Running
-Use `./silencebot -token <YOUR_TELEGRAM_TOKEN>` to start the bot.
+Use `./silencebot -t <YOUR_TELEGRAM_TOKEN>` to start the bot.
 
 By default it connects to localhost:6379 Redis instance without password and selects db 0.
 You can customize this behavior, check `./silencebot -h` for all arguments.
 
 ## Running as a background service
-It's up to you how you achieve that. I'm too lazy to make up any dockerfiles/etc :>
+It's up to you how you achieve that. ~~I'm too lazy to make up any dockerfiles/etc :>~~
```
Comment (stek29) [inline README.md]: Why won't you just remove this line?

File: main.go [hunk 3]
```diff
@@ -82,24 +89,28 @@ func checkMessage(m *tb.Message) {
...
 		db.Del(getAdminsKey(m.Chat))
 		return
 	}
 
 	if isSilent(m.Chat) {
 		setSilent(m.Chat, false)
+		log.WithField("chatID", m.Chat.ID).Debug("disabling silent mode")
 		go unrestrictAll(m.Chat)
-		b.Send(m.Chat, "*Режим тишины отключен. Можете общаться дальше.*", &tb.SendOptions{
+		b.Send(m.Chat, "*Silence mode disabled. Make flood great again.*", &tb.SendOptions{
```
Comment (stek29) [inline main.go:106]: Maybe messages should still be in Russian until localizations are implemented?

Comment (anna-oake) [reply main.go:106]: Actually I agree English is better at now.

File: README.md (hunk 2 above)
Comment (kirillDanshin) [reply README.md]: I leave it just for fun

**Pull request merged**